src/schema/types/user.graphql

```diff
@@ -0,0 +1,25 @@
+type User {
+  id: ID!
+  firstname: String!
+  lastname: String!
+  username: String!
+  salt: String
+  password: String!
+}
+
+extend type Query{
+  users: [User!]!
+  user (id: ID!): User
+}
+
+extend type Mutation {
+  createUser(user: UserInput): User!
+  deleteUser(id: ID!): Int
```

Comment:
Is it useful to return the `Int` parameter from the database here? I think from a GraphQL perspective, it's better to return a `Boolean` or even nothing at all (since GraphQL supports errors and a query/mutation can fail from a client perspective).